src/west/app/project.py

```diff
@@ -287,7 +287,7 @@ def local(self, args) -> Path:
         os.chdir(topdir)
         self.config = Configuration(topdir=topdir)
         self.config.set('manifest.path', os.fspath(rel_manifest))
-        self.config.set('manifest.file', manifest_filename)
+        self.config.set('manifest.file', manifest_file.name)
```

Comment:
This doesn't allow a nested manifest file, I think? It should be relative to the manifest path.